### PR TITLE
munin-murmur.py: Python3 and debian buster compatibility

### PR DESCRIPTION
**Pull request by brknkfr**

### CODE REVIEW

**Review by Krzmbrzl — APPROVED** [review]
Appears fine to me

File: Monitoring/munin-murmur.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8
 #
 # munin-murmur.py
-# Copyright (c) 2010 - 2016, Natenom <natenom@natenom.com>
+# Copyright (c) 2010 - 2016, Natenom
```
Comment (Kissaki) [inline Monitoring/munin-murmur.py:5]: How come there’s a change here in the copyright notice?

You could add yourself here too as you made changes here now.

**Review by Kissaki — APPROVED** [review]
Looks plausible.

File: Monitoring/munin-murmur.py (hunk 1 above)
Comment (brknkfr) [reply Monitoring/munin-murmur.py:5]: Oh, I didn't want to change that.

**Pull request merged**